tests/test_send_recv_two_workers.py

```diff
@@ -217,16 +213,14 @@ def cupy():
     not more_than_two_gpus(), reason="Machine does not have more than two GPUs"
 )
 @pytest.mark.parametrize(
-    "cuda_obj_generator", [dataframe, empty_dataframe, series, cupy]
+    "cuda_obj_generator", [dataframe, empty_dataframe, series, cupy_obj]
 )
 def test_send_recv_cu(cuda_obj_generator):
     base_env = os.environ
-    env1 = base_env.copy()
-    env2 = base_env.copy()
-    cvd = get_cuda_visible_devices()
+    env_client = base_env.copy()
+    cvd = get_cuda_visible_devices()[:3]
```

Comment:
Why `[:3]`?

Reply:
Yeah, that's not great.  The `[:3]` is because we are getting a string back from `get_cuda_visible_devices` function: `1,2,3,4,5,6,7`.  We only need the first device `1,2`

Reply:
I see you changed that to `[:2]` now, which seems to be what you wanted.